cdk/bin/api.ts

```diff
@@ -6,12 +6,16 @@ import { VpcStack } from "../lib/vpc-stack";
 import { RDSStack } from "../lib/rds-stack";
 
 const app = new cdk.App();
+const rdsPasswordArnSsmParamName = "rds-password-secret-arn" 
```

Comment:
Ah nice that way we don't need to define it multiple times. Do you mind copying the arn from here https://github.com/austinloveless/awsmug-serverless-graphql-api/blob/master/cdk/lib/api-stack.ts#L22 and added it on line 9

Reply:
I'm not a big fan of the hard coded arn, also that doesnt work outside of your account